icecreamshop/src/test/java/com/summerschool/icecreamshop/controller/BasketControllerTest.java

```diff
@@ -49,6 +91,19 @@ public void setUp() {
         basketDTO.setSessionId("ceva");
     }
 
+    @Test
+    public void testGetAllProducts(){
+        Mockito.when(basketService.getProductsFromBasket(2L))
+                .thenReturn(basketProductList);
+
+        Mockito.when(modelMapper.map(basket1, BasketProduct.class))
+                .thenReturn(basketProductMock);
+
+        ResponseEntity<List<BasketProductDTO>> response = basketController.getBasketProducts(1L);
+
+        assertEquals(HttpStatus.OK, response.getStatusCode());
```

Comment:
mai adauga un assert si specific continutului, ar trebui sa vezi ceva in neregula dupa asta.